app.rb

```diff
@@ -1,9 +1,25 @@
 require 'sinatra/base'
+require 'sinatra/reloader'
+require './lib/peeps'
```

Comment:
This all looks great!  The only thing I would suggest is that you revisit RESTful routing.  It's a really useful convention to follow and your future colleagues will probably expect you to do so.

The restful routes for this app would be

```
get '/peeps' # to get all the peeps and render them in the :index view
get '/peeps/new' # to get the form which is used to create a new peep
post '/peeps' # to handle form submission
```